plugin/storage/es/options.go

```diff
@@ -60,12 +64,16 @@ func NewOptions(primaryNamespace string, otherNamespaces ...string) *Options {
 	options := &Options{
 		primary: &namespaceConfig{
 			Configuration: config.Configuration{
-				Username:    "",
-				Password:    "",
-				Sniffer:     false,
-				MaxSpanAge:  72 * time.Hour,
-				NumShards:   5,
-				NumReplicas: 1,
+				Username:          "",
+				Password:          "",
+				Sniffer:           false,
+				MaxSpanAge:        72 * time.Hour,
+				NumShards:         5,
+				NumReplicas:       1,
+				BulkSize:          5 * 1000 * 1000,
+				BulkWorkers:       1,
+				BulkActions:       1000,
+				BulkFlushInterval: 0,
```

Comment:
Maybe set the default to 100ms? 
I think that with 0 spans might not be flushed if someone is just testing and sends less than 1000 spans. Or the added delay will be high at low throughputs.

Reply:
100ms seems quite low to leverage bulk API. If somebody is testing even 1s could be fine, until you open a browser and refresh spans will be there. 

Here are some tests, flush interval: test duration
100ms: 100s
200ms: 80s
300ms: 77s
500ms: ~76s
1s: ~75s 

I would go with at least 200ms